portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -2033,6 +2370,150 @@ definition {
 		}
 	}
 
+	@description = "LPS-122873 When any web content is unassigned to a Vocab1's category, the relative ordering of categories remain the same"
+	@priority = "5"
+	test SlelectSomeCategoriesFromVocab1 {
```

Comment:
Change the title of this test, is not easy to know what it does. Dont refer it to concrete names. Also it has a mistake in the Select